src/types/common.d.ts

```diff
@@ -0,0 +1,9 @@
+type PaginatedUsersCompose = {
+    lastEvaluatedUser: Partial<User>
+    users: any//IteratorResult<User[], any>
```

Comment:
What are the pros and cons of using `any`, and why use it here?

Reply:
Pros: It would be easier to return any type in this attribute.

Cons: I would be not enforcing an specific type, so it would be as I were using plain javascript.

The reason I used, it was because I was testing the type of this attribute that was returned from the Scanner and I forget to uncomment the code you see in the right hand